projects/plugins/jetpack/extensions/extended-blocks/index.scss

```diff
@@ -0,0 +1,28 @@
+.is-beta-extension {
+	position: relative;
+
+	&::before {
+		content: 'BETA Extension';
```

Comment:
This may be problematic, since it cannot be translated that way.

Reply:
yeah... let me rename it as BETA

Reply:
90a2c7995b288cf1a1addeebf494b627c344bdff